packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -152,7 +144,7 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
         ...getFilledColorProps(
           {
             backgroundColor: globalTokens.color.yellow.primary,
-            color: globalTokens.color.grey[14],
+            color: t.colors.neutralForeground1,
```

Comment:
> color: t.colors.neutralForeground1, [](http://example.com/codeflow?start=11&length=36)

This is neutralForegroundStatic1, which is different from neutralForeground1. Unfortunately I don't have this alias token in win32 since it wasn't specified when I created the tokens, so I will need to add that to the token set for win32. In the meantime you should hardcode the color to grey[14], since that's the intended color for this alias token on both light and dark modes.